docs/src/changelog.md

```diff
@@ -215,7 +216,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ### Added
 
 - When using Jira as source for the 'issues' and the 'user story points' metric, show the issue type in the metric details. Closes [#1674](https://github.com/ICTU/quality-time/issues/1674).
-- Allow for limiting editing rights to specific people. Grant editing rights to people by adding their username or email address to the editors field on the homepage. Expand the overview title to access the editors field. Also see the [user manual](USAGE.md#limiting-editing-rights). Closes [#294](https://github.com/ICTU/quality-time/issues/294).
+- Allow for limiting editing rights to specific people. Grant editing rights to people by adding their username or email address to the editors field on the homepage. Expand the overview title to access the editors field. Also see [](usage.md#configuring-permissions). Closes [#294](https://github.com/ICTU/quality-time/issues/294).
```

Comment:
a link without text, is it still clickable?

Reply:
It works in VS Code, not on GitHub unfortunately. The advantage of this way of linking is that Sphinx uses the section title from the linked section to create the link in the document; less duplication. But seeing it doesn't work on GitHub, I'll add a link text.